tests/filter-tests.js

```diff
@@ -148,6 +148,7 @@ describe('broccoli-asset-rev', function() {
     var node = new AssetRewrite(sourcePath + '/input', {
       replaceExtensions: ['js'],
       assetMap: {
+        'images/some-image.jpg': 'images/some-image-1a2b3c4e.jpg',
         'the.map' : 'the-other-map',
         'http://absolute.com/source.map' : 'http://cdn.absolute.com/other-map'
```

Comment:
Nice. I'll also check how my algorithm behaves in with relative paths in css files. I'm not sure what the behaviour should be, since if the stylesheet gets a host prepended, the images referenced by it can continue to be relative, because iirc they use the host of the stylesheet